C4SGSvc/src/main/java/org/c4sg/constant/C4sgConstants.java

```diff
@@ -0,0 +1,11 @@
+package org.c4sg.constant;
+
+public class C4sgConstants {
```

Comment:
wait, see here, you need a more specific name for your constant class
![image](https://cloud.githubusercontent.com/assets/24813620/23736559/b1b456e0-0452-11e7-965b-e57ba4e7907c.png)